src/pages/Landing.jsx

```diff
@@ -103,10 +105,12 @@ function Landing({ classes }) {
         <Container maxWidth="lg">
           <Logo color="white" />
           <div className={classes.authBtnContainer}>
-            <Link to="/login">Log In</Link>
-            <Link to="/signup">
+            <MUILink underline="none" onClick={() => loginWithRedirect()}>
+              Log In
+            </MUILink>
```

Comment:
I forgot to remove this `onClick` callback... The function doesn't exist in the file, so clicking it will produce an error at the moment. I was planning on adding it in the next auth PR.